hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/TotalTargetCountStatus.java

```diff
@@ -154,8 +159,7 @@ private void convertStatus(final TotalTargetCountActionStatus item) {
         case DOWNLOAD:
         case DOWNLOADED:
         case CANCELING:
-            final Long runningItemsCount = statusTotalCountMap.get(Status.RUNNING) + item.getCount();
-            statusTotalCountMap.put(Status.RUNNING, runningItemsCount);
+            mapItemStatus(item);
```

Comment:
check if mapping all these statuses to FINISHED in case of DOWNLOAD_ONLY is correct